bin/asinit

```diff
@@ -377,9 +377,14 @@ function ensurePackageJson() {
 function ensureIndexJs() {
   console.log("- Making sure that 'index.js' exists...");
   if (!fs.existsSync(indexFile)) {
+    // since node.js v13.2.0 or v12.17.0 we can use ESM without flags
+    const ver = process.versions.node.split('.');
+    const maj = parseInt(ver[0]);
+    const min = parseInt(ver[1]);
+    const supportESM = (maj >= 12 && min >= 17 && maj < 13) || (maj >= 13 && min >= 2);
```

Comment:
Does this cover versions like 14.0?

Reply:
It should. It covers all  from`12.17` to `13.0` (exclusive) and from `13.2` to `latest`

Reply:
```
> maj = 14
14
> min = 0
0
> (maj >= 12 && min >= 17 && maj < 13) || (maj >= 13 && min >= 2)
false
```

```
> maj >= 14 || (maj == 12 && min >= 17) || (maj == 13 && min >= 2)
true
```

Reply:
right! Will fix this